app/export.go

```diff
@@ -338,3 +344,90 @@ func (a *App) BuildPostReactions(postId string) (*[]ReactionImportData, *model.A
 	return &reactionsOfPost, nil
 
 }
+
+func (a *App) ExportCustomEmoji(writer io.Writer, file string) *model.AppError {
```

Comment:
can we add a unit test for this one?

Reply:
I was wondering on how to write a better test case for `ExportCustomEmoji ` method.

Right now we have got a variable with value as `emojiImagePath := "data/emoji/" + emoji.Id + "/image"` in `ExportCustomEmoji` method. Here we got a limitation that the bulk export command should be run from the root of the repo.

The test case we write will be run with the command `go test -v -run='TestExportCustomEmoji' ./app` - not finding `data` dir inside `app` dir, we'll get errors.

One solution for this is initialise  `emojiImagePath`  in `commands/exports.go` and pass is as argument as `a.BulkExport(fileWriter, args[0], emojiImagePath)`. This will allow us to pass `emojiImagePath` from test case as well.